csrankings-k.csv

```diff
@@ -74,12 +74,14 @@ Kaigui Bian,Peking University,http://net.pku.edu.cn/~bkg,P8WKqa4AAAAJ
 Kaihuai Qin,Tsinghua University,http://www.tsinghua.edu.cn/publish/cs/4616/2011/20110907090317797876810/20110907090317797876810_.html,NOSCHOLARPAGE
 Kaili Müürisep,University of Tartu,http://kodu.ut.ee/~kaili,BSsD0vgAAAAJ
 Kaili Zhao,BUPT,https://zkl20061823.github.io,kRbj_0kAAAAJ
+Kailun Yang 0001,Hunan University,https://yangkailun.com,pKFqWhgAAAAJ
 Kaiming Shen,CUHK (SZ),https://myweb.cuhk.edu.cn/shenkaiming,ZXex-IkAAAAJ
 Kaiqi Huang,Chinese Academy of Sciences,http://people.ucas.ac.cn/~huangkaiqi?language=en,caQ-OmYAAAAJ
 Kaisa Väänänen,Tampere University,https://research.tuni.fi/kaisa-vaananen,mt41HYQAAAAJ
 Kaishun Wu,Shenzhen University,http://www.kaishunwu.com/index.html,Vt-q3E4AAAAJ
 Kaitai Liang,TU Delft,https://www.tudelft.nl/ewi/over-de-faculteit/afdelingen/intelligent-systems/cybersecurity/computational-privacy/people/kaitai-liang,_qHKRH4AAAAJ
 Kaito Fujii,National Institute of Informatics,https://www.nii.ac.jp/en/faculty/informatics/fujii_kaito/,RcFjdYwAAAAJ
+Kaiwei Wang,Zhejiang University,http://wangkaiwei.org/indexeg.html,B6xWNvgAAAAJ

```

Comment:
Not in CS. Please clarify how they meet the inclusion criteria.

Reply:
Prof. Kaiwei Wang is a full professor, who advises Ph.D. students in Information Sensing and Instrumentation, a combination of computer science (informatics) and optics. So Prof. Kaiwei Wang meets the inclusion criteria.

Reply:
That is not an accurate reading of the requirements. They must be able to solely advise PhD students for a degree _in Computer Science_.

Reply:
Prof. Kaiwei Wang solely advises Ph.D. students for a degree in Information Sensing and Instrumentation.
https://person.zju.edu.cn/en/0009214

If this does not meet the inclusion criteria, then please leave Prof. Kaiwei Wang out and only include Prof. Kailun Yang.

Thanks.

Reply:
Should be removed. Please remove this entry.

Reply:
It has been removed. Thanks.